### DISCUSSION TITLE
Help Setting up Custom VPN on my Server

### DISCUSSION THREAD

**Discussion by waltuy** [qdm12/gluetun#460] [Ideas]
Hello Everyone,
I use [Cloudbox](https://github.com/Cloudbox/Cloudbox) media server which is based on docker, I am trying to add gluetun container on it so I can route multiple containers through gluetun by using only 1 VPN connection. A friend of mine used below setup to successfully setup gluetun on his Cloudbox system (he uses Surfshark VPN which is supported by gluetun). I am using ExpressVPN provider which is not supported by gluetun so I am trying a custom setup.

My & my friends setup are shown below, I am wondering what should I input in -e VPNSP=  & -e REGION= for my custom vpn setup.? or any other suggestions 

This is my custom.conf fiile:- [custom.txt](https://github.com/qdm12/gluetun/files/6529419/custom.txt)

I am following the steps as noted here: [https://github.com/qdm12/gluetun/issues/223#issuecomment-791994363](https://github.com/qdm12/gluetun/issues/223#issuecomment-791994363) & [https://github.com/qdm12/gluetun/wiki/Openvpn-file](https://github.com/qdm12/gluetun/wiki/Openvpn-file)

Thanks in advance for your time and help.

```
My Friends Setup below: He uses Surfshark

docker run -d  \
    --name=gluetun  \
    --cap-add=NET_ADMIN \
    --restart=unless-stopped  \
    -e PGID=1000 -e PUID=1000  \
    -e VPNSP="surfshark" \
    -e REGION="Switzerland" \
    -e OPENVPN_USER=xxxxx \
    -e OPENVPN_PASSWORD=xxxxx \
    -v /opt/gluetun:/config  \
    -v /etc/localtime:/etc/localtime:ro  \
    --label com.github.cloudbox.cloudbox_managed=true \
    --network=cloudbox  \
    --network-alias=gluetun  \
    qmcgaw/gluetun
```

```
My Setup below: I use ExpressVPN

docker run -d  \
    --name=gluetun  \
    --cap-add=NET_ADMIN \
    --restart=unless-stopped  \
    -e PGID=1000 -e PUID=1000  \
    -e VPNSP="???" \
    -e REGION="???" \
    -e OPENVPN_USER=xxxxx \
    -e OPENVPN_PASSWORD=xxxxx \
    -e OPENVPN_CUSTOM_CONFIG=/opt/gluetun/custom.conf \
    -v /opt/gluetun:/config  \
    -v /etc/localtime:/etc/localtime:ro  \
    --label com.github.cloudbox.cloudbox_managed=true \
    --network=cloudbox  \
    --network-alias=gluetun  \
    qmcgaw/gluetun:ovpn-file
```

**Comment by qdm12** [comment]
Just don't set values such as `VPNSP`. As long as you set `OPENVPN_CUSTOM_CONFIG` it should be working. Note however the value you have here seems wrong, it should be `/config/custom.conf` as the program only knows its container file system and not the host (check your volume bind mount)
